challenges-webapp/src/main/webapp/vue-app/challenges/components/AnnouncementDrawer.vue

```diff
@@ -101,27 +109,32 @@ export default {
   data() {
     return {
       announcement: {},
-      isValidDescription: {
-        description: true },
+      templateParams: {},
+      validInput: true,
+      MAX_LENGTH: 1300,
     };
   },
   computed: {
     space() {
       return this.challenge && this.challenge.space;
     },
     disabledSave() {
-      return this.announcement.assignee && this.announcement.assignee.length > 0 && this.isValidDescription.description ;
+      return this.announcement.assignee && this.announcement.assignee.length > 0 && this.validInput && this.announcement.comment && this.announcement.comment.length > 0;
```

Comment:
we may indicate to the end user what field is mandatory and must be filled, else, we will have a very bad UX